tests/unit/test_specfile.py

```diff
@@ -0,0 +1,57 @@
+# Copyright Contributors to the Packit project.
+# SPDX-License-Identifier: MIT
+from packit.specfile import Specfile
+
+
+def test_set_spec_has_autochangelog(tmp_path):
```

Comment:
Try using `pytest.mark.parametrize` to break up this test into 3 separate ones.

Written like this, one wouldn't get the result of the last 2 asserts if the first one fails, even though they are independent of each other.